lib/omnisharp-atom/linter.ts

```diff
@@ -84,6 +84,13 @@ class LinterCSharp extends Linter.Linter {
             });
         }
     }
+
+    private getFrameworks(error: OmniSharp.Models.DiagnosticLocation): string {
+        var frameworks = _.map(error.Projects, (project: string) => {
+            return project.split('+')[1];
+        }).filter((fw: string) => fw.length > 0);
+        return frameworks.join(',');
+    }
```

Comment:
I was thinking of putting it on Omni but don't want to populate Omni with too much as well. Maybe we should create a utils class?